linz-data-importer/about.html

```diff
@@ -0,0 +1,27 @@
+<h1 id="linz-data-importer">LINZ Data Importer</h1>
+<p>This plugin allows you to easily discover and import data from Toitū Te Whenua Land Information New Zealand into QGIS.</p>
+<p>Explore and connect to LINZ's WMTS and WFS services from within QGIS.</p>
+
+<h2>Available data</h2>
+<p>Data (available through WFS or WMTS) from the agency data portals and services below are available via the LINZ Data Importer.</p>
+
+<h3>Open data portals</h3>
+<ul>
+    <li>Toitū Te Whenua – Land Information New Zealand (<a href="https://data.linz.govt.nz">https://data.linz.govt.nz</a>)</li>
+    <li>Manatū Mō Te Taiao – Ministry for the Environment (<a href="https://data.mfe.govt.nz">https://data.mfe.govt.nz</a>)</li>
+    <li>Tatauranga Aotearoa – Statistics New Zealand (<a href="https://datafinder.stats.govt.nz">https://datafinder.stats.govt.nz</a>)</li>
+    <li>Manaaki Whenua – Landcare Research (<a href="https://lris.scinfo.org.nz">https://lris.scinfo.org.nz</a>)</li>
+    <li>Te Ope Kātua o Aotearoa – New Zealand Defence Force (<a href="https://geodata.nzdf.mil.nz">https://geodata.nzdf.mil.nz</a>)</li>
+</ul>
+
+<h3>Basemaps services</h3>
+<ul>
+    <li>LINZ Basemaps (<a href="https://basemaps.linz.govt.nz">https://basemaps.linz.govt.nz</a>)</li>
+</ul>
+
+<p>The plugin supports data available via WMTS and WFS services. If you are unsure which service best suits your needs, see the resources featured in the guides and documentation section of the <a href="https://www.linz.govt.nz/">LINZ website</a>.</p>
+
+<h2>Get started</h2>
+
+<p>The first step in using the LINZ Data Importer is to get an API key. An API key is required from each service you wish to access data from via the plugin.</p>
+<p>See the Help tab for details on how to generate an API keys.</p>
```

Comment:
It would be good to add newlines at EOF for all the text files; Git expects it, which is why it's showing the stop sign at the end of the last line here.